centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -24605,3 +24605,95 @@ msgstr "alerte"
 #: unknown
 msgid "will display resources with host name starting by'"
 msgstr "affichera les ressources avec un nom d'hôte commençant par"
+
+msgid "Welcome to the Commands page"
+msgstr "Bienvenue sur la page Commandes"
+
+msgid "Commands are used by the monitoring engine to run plugins to perform actions. While most commands come ready-to-use via Monitoring Connectors, you can also create custom commands for specific needs."
+msgstr "Les commandes sont utilisées par le moteur de surveillance pour exécuter des plugins afin d'effectuer des actions. Bien que la plupart des commandes soient prêtes à l'emploi via les connecteurs de surveillance, vous pouvez également créer des commandes personnalisées pour des besoins spécifiques."
+
+msgid "Service uses"
```

Comment:
I corrected the English based on the French, but I'd like to see the label in the interface to be sure what it does.